tasks/shell_crmsh/cluster-destroy-crm.yml

```diff
@@ -23,6 +27,7 @@
     - pacemaker
     - corosync
     - corosync-qdevice
+  when: item in ansible_facts.services
```

Comment:
Are you sure that this works? On my systems, service names in `ansible_facts.services` have `.service` suffix.

Reply:
@tomjelinek Ansible service_facts documentation shows that variable is ansible_facts.services
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_facts_module.html

Reply:
@tomjelinek means that service names should be suffixed with .service, like `pacemaker.service`, `corosync.service`, `corosync-qdevice.service`.

Reply:
@spetrosi @tomjelinek Thank you for clarification, I have updated service names with .service and retested.